trio/_core/_generated_io_kqueue.py

```diff
@@ -12,9 +12,8 @@
     import select
     from socket import socket
 
-    from ._traps import Abort, RaiseCancelT
-
     from .. import _core
+    from ._traps import Abort, RaiseCancelT
```

Comment:
Hmm, I didn't see an edit to the script that generates these files -- please edit that!

Reply:
Slightly confused about your wording here, I'm not sure what you would like me to do.

Reply:
This file is generated, you'll need to update `gen_exports.py` to change the imports.

Reply:
I thought I already fixed that with the "run ruff now instead of isort" change

Reply:
Anyways, with 9cf4394a it changes the kqueue import string to be what it will after ruff runs.